galaxy_importer/constants.py

```diff
@@ -73,6 +83,8 @@ class ContentType(enum.Enum):
     TERMINAL_PLUGIN = "terminal"
     TEST_PLUGIN = "test"
     VARS_PLUGIN = "vars"
+    EDA_EVENT_SOURCE = EDA_EVENT_SOURCE_NAME
```

Comment:
Should EDA_EVENT_SOURCE = "event_source" and EDA_EVENT_FILTER = "event_filter" to indicate the type instead of being the full path?

Reply:
In the future with custom extensions being used in core (and maybe being allowed by galaxy-importer) both `eda/plugins/event_sources` and `my_thing/plugins/event_sources` could exist. Also per core this value should be considered the plugin type, and that is what will be used with `ansible-doc -t`... `eda/plugins/event_sources` just like `module_utils` or `shell`. If the web UI display is desired to be different, galaxy can eventually utilize the `display_name` in the spec for `meta/extensions.yml`

Reply:
Got it!